_posts/2019-04-23-chat-about-tpu-5.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "闲话TPU #5 Goole TPU 线下分享交流会"
+categories: "TPU Blog"
+author: "Jun.Li"
+date: 2019-04-23
+---
+
+版权声明：本博文欢迎分享与转载，转载请注明出处和作者。<lj120118@163.com>
+
+> [闲话TPU #1 背景/价格/TFRC计划及羊毛](http://cyfeng.science/tpu/blog/2019/03/07/chat-about-tpu-1.html)
+>
+> [闲话TPU #2 配置GCP环境/创建TPU实例](http://cyfeng.science/tpu/blog/2019/03/06/chat-about-tpu-2.html)
+>
+> [闲话TPU #3 模型编写](http://cyfeng.science/tpu/blog/2019/03/05/chat-about-tpu-3.html)
+> 
+> [闲话TPU #4 Coral Edge TPU赋能移动端](http://cyfeng.science/tpu/blog/2019/03/04/chat-about-tpu-4.html)
+
+第一届Google TPU线下分享交流会于2019年4月19日在中关村五道口高山会议室成功举办。参会人员分别为[投资人兼在读博士李卓桓](https://bkso.baidu.com/item/%E6%9D%8E%E5%8D%93%E6%A1%93)、[SIGAI创始人雷明](https://www.huodongjia.com/guest-077106065120077122081053.html)、[伽利略资本投资总监郑譞](https://cn.linkedin.com/in/xuanr) [奇点机智职员栾迎凯](https://www.naturali.io/about-us)、北邮硕士[段嘉铭](https://github.com/VDeamoV)、[朱正源](https://github.com/824zzy)、[李童俊](https://github.com/TuringLee)、[石珅达](https://github.com/CyFeng16)，中科院电子所博士生[李军](https://github.com/muzi-8)
+
+![AI at the edge](../../../../../assets/TPU/photo.jpg)
```

Comment:
1. Do NOT use UPPER CASE in filename/directory;
1. Use the same filename as the blog, and put the picture directly in the year directory, like `2019/`

Please use `/assets/2019/chat-about-tpu-xxx.jpg` as your photo name.